tableau_utilities/tableau_file/tableau_file_objects.py

```diff
@@ -715,12 +723,14 @@ def dict(self):
 
 @dataclass
 class Refresh(TableauFileObject):
-    increment_key: str
-    incremental_updates: bool
     tag: str = 'refresh'
+    refresh_event: dict = None
```

Comment:
@JustinGrilli  this fixed the errors I was seeing.  I'm not sure why some Tableau files have this and some don't. I would guess it's a version issue and newer .tdsx files have this.  

```
   <refresh>
            <refresh-event add-from-file-path='Infrastructure Costs (local copy)' increment-value='%null%' refresh-type='create' rows-inserted='42729621' timestamp-start='2022-12-27 23:45:52.459' />
          </refresh>
```

Reply:
Yeah Tableau probably made some updated to the XML lately.
Makes sense, thanks for updating!